src/services/forest-server-requester.js

```diff
@@ -5,30 +5,35 @@ const errorMessages = require('../utils/error-messages');
 const VError = require('verror');
 
 function ForestServerRequester() {
-  this.perform = (route, environmentSecret) => {
+  this.perform = (route, environmentSecret, queryParameters) => {
     const urlService = new ServiceUrlGetter().perform();
 
     return new P((resolve, reject) => {
-      request
+      const currentRequest = request
```

Comment:
I think you should do some renaming:
- `request` => `superagent`
- `currentRequest` => `request`